docs/en/latest/configuration/environment-variables.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Environment Variables
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Default Environment Variables
+
+Many environment variables can be used to configure APISIX and change its behavior. Not all the environment variables are user-MODIFICABLE.
+
+:::note
+Environment variables beginning with `APISIX_` are reserved for APISIX internal usage. Do not define any environment variables with this prefix or modify them unless advised to do so.
+
+:::
+
+#### `APISIX_CONF_EXPIRE_TIME`
+
+| Default | Type            | Remarks                                            |
+| ------- | --------------- | -------------------------------------------------- |
+| 3600s   | NON-MODIFICABLE | Only used in the multilingual plugins (ext-plugin) |
+
+When the APISIX configuration is modified, it sends a new `PrepareConf` call to the Plugin Runner. Currently, no mechanism exists to notify the Plugin Runner of configuration change/removal. The `APISIX_CONF_EXPIRE_TIME` is a workaround to set the conf cache to expire time. The Plugin Runner caches the conf slightly longer than this environment variable value, if the configuration is still existing after the set time, a new `PrepareConf` call is sent to create a new configuration.
+
+#### `APISIX_PATH`
```

Comment:
if this doesn't need to explain, so we can ignore it?

Reply:
Yeah, we can, but it will be nice to present as much information we have in the docs.

Reply:
what about listing them in a unified place, rather than occupying a position without any meaning?

Reply:
Okay this will be better

Reply:
Done